developers/integrations/compute-infrastructure/modal/index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Modal
+sidebar_position: 1
+---
+
+[Modal](https://modal.com/) provides a serverless platform that enables developers to easily deploy and automatically scale applications with on-demand access to GPUs and a custom high-performance container runtime.
+
+## Modal and Weaviate
+Weaviate integrates with Modal by leveraging its serverless infrastructure for the fast generation of embeddings and generative model calls. The [Weaviate client can be hosted](https://modal.com/docs/examples/vector-analogies-wikipedia#deploy-a-serverless-read-only-weaviate-client-with-modal) on Modal to dynamically scale based on workload demands. 
+
+
+## Our Resources 
+**Hands on Learning**: Content framed to build your technical understanding with end-to-end tutorials.
```

Comment:
Consider shortening:
Build your technical understanding with end-to-end tutorials.